src/Uno.UI/UI/Xaml/Controls/Control/Control.cs

```diff
@@ -990,152 +1140,6 @@ protected virtual void OnBorderBrushChanged(Brush oldValue, Brush newValue)
 		private static readonly Type[] _manipInertiaArgsType = new[] { typeof(ManipulationInertiaStartingRoutedEventArgs) };
 		private static readonly Type[] _manipCompletedArgsType = new[] { typeof(ManipulationCompletedRoutedEventArgs) };
 
-		protected static RoutedEventFlag GetImplementedRoutedEvents(Type type)
```

Comment:
This is going to be a breaking change :/
Even if the method is kept, it will have to be an instance virtual method.

Reply:
It was designed to be used for generated code eventually...  We just followed another path.

Reply:
@carldebilly Do you mean a breaking change would be okay here?

Reply:
I can't imagine any case where client code could implement or call those...  So I don't see any problem removing this method.